tests/postgres/vm.test.ts

```diff
@@ -76,6 +81,29 @@ test('setupPlugin async', async () => {
     expect(newEvent.event).toEqual('haha')
 })
 
+test('teardownPlugin', async () => {
+    const indexJs = `
+        function setupPlugin (meta) {
+            meta.global.data = 'haha'
+        }
+        function teardownPlugin (meta) {
+            fetch('https://google.com/results.json?query=' + meta.global.data)
+        }
+        function processEvent (event, meta) {
+            meta.global.data = event.properties.haha
+            return event
+        }
+    `
+    await resetTestDatabase(indexJs)
+    const vm = await createPluginConfigVM(mockServer, pluginConfig39, indexJs)
+    await vm.methods.processEvent({
+        ...defaultEvent,
+        properties: { haha: 'hoho' },
+    })
+    await vm.methods.teardownPlugin()
```

Comment:
Note: Probably worth adding `expect(fetch).not.toHaveBeenCalled()` above this line

Reply:
👍